charts/memgraph-platform/README.md

```diff
@@ -0,0 +1,76 @@
+| `service.portLab`                            | Service port for Lab UI                                     | `3000`                      |
+| `service.portLog`                            | Service port for logs                                       | `7444`                      |
+| `env.MEMGRAPH`                               | Memgraph environment variable                               | `{}`                        |
+| `persistentVolumeClaim.storagePVC`           | Enable persistent volume claim for storage                  | `true`                      |
+| `persistentVolumeClaim.storagePVCSize`       | Size of the persistent volume claim for storage             | `1Gi`                       |
+| `persistentVolumeClaim.logPVC`               | Enable persistent volume claim for logs                     | `true`                      |
+| `persistentVolumeClaim.logPVCSize`           | Size of the persistent volume claim for logs                | `256Mi`                     |
+| `ingress.enabled`                            | Enable Ingress                                              | `false`                     |
+| `ingress.className`                          | Ingress class name                                          | `""`                        |
+| `ingress.annotations`                        | Ingress annotations                                         | `{}`                        |
+| `ingress.hosts`                              | Ingress hosts                                               | `[]`                        |
+| `resources`                                  | Resource requests and limits                                | `{}`                        |
+| `livenessProbe`                              | Liveness probe settings                                     | `{}`                        |
+| `readinessProbe`                             | Readiness probe settings                                    | `{}`                        |
+| `autoscaling.enabled`                        | Enable Horizontal Pod Autoscaler                            | `false`                     |
+| `autoscaling.minReplicas`                    | Minimum number of replicas for autoscaling                  | `1`                         |
+| `autoscaling.maxReplicas`                    | Maximum number of replicas for autoscaling                  | `100`                       |
+| `autoscaling.targetCPUUtilizationPercentage` | Target CPU utilization percentage for autoscaling           | `80`                        |
+| `volumes`                                    | Additional volumes on the output Deployment definition      | `[]`                        |
+| `volumeMounts`                               | Additional volumeMounts on the output Deployment definition | `[]`                        |
+| `nodeSelector`                               | Node selector for pods                                      | `{}`                        |
```

Comment:
`nodeSelector`, `tolerations` and `affinity` doesn't seem to be used in the chart.